framework/base/ExitException.php

```diff
@@ -33,6 +33,8 @@ class ExitException extends \Exception
     public function __construct($status = 0, $message = null, $code = 0, $previous = null)
     {
         $this->statusCode = $status;
+        $message  = isset($message) ? $message : '';
+        $previous = isset($previous) ? $previous : new \Exception();
         parent::__construct($message, $code, $previous);
```

Comment:
```suggestion
        parent::__construct((string)$message, $code, $previous);
```

Did `$previous` stopped to accept `null`?

Reply:
in this case, the $previous requires an instance of Throwable so I use the `new \Exception()`

Reply:
Then it's better to do:

```php
if ($previous === null) {
    parent::__construct((string)$message, $code);
} else {
    parent::__construct((string)$message, $code, $previous);
}
```